cli.js

```diff
@@ -102,25 +104,20 @@ engine(
     extensions: extensions,
     configTransform: transform,
     output: false,
-    out: false,
-    streamError: new PassThrough(),
     rcName: '.alexrc',
     packageField: 'alex',
+    color: supportsColor.stdout,
```

Comment:
```suggestion
    color: supportsColor.stderr,
```

Not sure if this works, but the report is written to stderr, so checking whether that has color makes more sense?

Reply:
yea that makes sense. I had to add this because colours disappeared after the refactor. I've changed it